### PR TITLE
[Feature] new aggregator Jumper for withdrawals widget

### PR DESCRIPTION
**Pull request by Jeday**
### Description

- added new dex Jumper
- updated styles to support >3 dexes
- did not enable it config yet

### Demo

image

[DEMO VIDEO](https://github.com/lidofinance/ethereum-staking-widget/assets/3705803/d8eea2f4-a0d4-4d39-83dc-bfb9dd51aa23)

### Testing notes

Can be checked with `mock-qa-helpers-enabled-withdrawal-dexes`=`['one-inch', 'paraswap', 'bebop','jumper']`
After #344 is merged, if this branch is updated, can be tested by overriding IPFS.json

### Checklist:

- [x] Checked the changes locally.
- [ ] Created / updated analytics events.
- [ ] Created / updated the technical documentation (README.md / [docs](https://docs.lido.fi/) / etc.).
- [ ] Affects / requires changes in other services (Matomo / Sentry / CloudFlare / etc.).

### CODE REVIEW

File: features/withdrawals/request/form/options/styles.tsx [hunk 1]
```diff
@@ -265,3 +320,30 @@ export const DexWarning = styled.div`
     margin-right: ${({ theme }) => theme.spaceMap.xs}px;
   }
 `;
+
+const DexOptionsCheckMark = (props: React.ComponentProps<'svg'>) => (
+  <svg
```
Comment (DiRaiks) [inline features/withdrawals/request/form/options/styles.tsx:325]: Might it make sense to move the svg to the icons folder?

Comment (Jeday) [reply features/withdrawals/request/form/options/styles.tsx:325]: I thought I might need to dynamic render this svg, but forgot to move to icons

Comment (Jeday) [reply features/withdrawals/request/form/options/styles.tsx]: fixed

File: features/withdrawals/request/form/options/dex-options/styles.ts [hunk 2]
```diff
@@ -0,0 +1,153 @@
+import styled from 'styled-components';
+import { InlineLoader, ThemeName } from '@lidofinance/lido-ui';
+import ExternalLink from 'assets/icons/external-link-icon.svg';
+import { ReactComponent as ChevronBlue } from 'assets/icons/chevron-blue.svg';
+
+export const DexOptionsContainer = styled.div<{
+  $maxElements: number;
+  $showMore: boolean;
```
Comment (DiRaiks) [inline features/withdrawals/request/form/options/dex-options/styles.ts:8]: It is not used here

File: features/withdrawals/request/form/options/dex-options/dex-options.tsx [hunk 3]
```diff
@@ -79,34 +79,37 @@ export const DexOptions: React.FC<
...
-              url={link(amount, selectedToken)}
-              key={title}
-              toReceive={rate ? toReceive : null}
-            />
-          );
-        })}
+    <>
+      <DexOptionsContainer
+        data-testid="dexOptionContainer"
+        $showMore={showMore}
```
Comment (DiRaiks) [inline features/withdrawals/request/form/options/dex-options/dex-options.tsx:86]: not nessary props

**Review by DiRaiks — CHANGES_REQUESTED** [review]

**Review by DiRaiks — APPROVED** [review]
lgtm

**Pull request merged**